app/assets/stylesheets/new-capture.css

```diff
@@ -0,0 +1,610 @@
+* {
```

Comment:
Let's de-duplicate this with https://github.com/publiclab/spectral-workbench.js/blob/main/examples/new-capture/new-capture.css - and name it differently so it's clear that it isn't redundant - so this file in this PR should have only those things that are specific to the Rails app here, and everything else should go upstream in the JS library, and then https://github.com/publiclab/spectral-workbench.js/blob/main/examples/new-capture/new-capture.css would be referenced in addition to the local version here from index2.html. Make sense?

Reply:
@waridrox i liked what you said about including both stylesheets in succession. Let's do that!